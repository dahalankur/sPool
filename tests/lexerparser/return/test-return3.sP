# A function with two returns is syntactically permitted
def store bool myFunc (int a, bool b):
    return a
    return a + 1;
