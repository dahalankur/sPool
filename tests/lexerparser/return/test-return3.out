PROGRAM {[ 
    FUN(STORE, BOOLTY, myFunc, FORMALS((INTTY, a), (BOOLTY, b)), [ 
        RETURN(VAR(a)) 
        RETURN(BINOP(VAR(a), PLUS, INT(1)))])]}
